test/cea/cea_decoder_unit.js

```diff
@@ -68,14 +58,15 @@ describe('AtscDecoder', () => {
 
       const topLevelCue = new shaka.text.Cue(
           startTimeCaption1, startTimeCaption2, '');
-      const nestedCue = new shaka.text.Cue(
-          startTimeCaption1, startTimeCaption2, expectedText);
-      nestedCue.color = 'green';
-      nestedCue.backgroundColor = 'black';
-      nestedCue.textDecoration.push(shaka.text.Cue.textDecoration.UNDERLINE);
+
+      const nestedCue = ceaUtils.createStyledCue(
+          startTimeCaption1, startTimeCaption2, expectedText,
+          true, false, 'green', 'black'
```

Comment:
I could use inline comments on true, false, and string literals to remind me what those parameters are.